src/Marten/StoreOptions.cs

```diff
@@ -95,6 +95,11 @@ public string DatabaseSchemaName
         public int UpdateBatchSize { get; set; } = 500;
 
         /// <summary>
+        /// Configures the store to use char buffer pooling, greatly reducing allocations for serializing documents and events.
+        /// </summary>
+        public bool UseCharBufferPooling { get; set; } = false;
```

Comment:
Should the default be to use char buffer pooling? I'm not a fan of having a `bool` default to true so perhaps consider flipping the name and logic around to something like `SuppressCharBufferPooling`.